backend/src/main/java/eliasyazdani/capstone/cupwithme/backend/player/PlayerWithOrdinalNumber.java

```diff
@@ -1,10 +1,10 @@
 package eliasyazdani.capstone.cupwithme.backend.player;
 
-public record PlayerWithOrdinalNumber (
-    int id,
-    String firstName,
-    String lastName,
-    int age
-){
+public record PlayerWithOrdinalNumber(
```

Comment:
Wozu ist diese Klasse? Mir scheint sie identisch mit `Player` zu sein.

Reply:
in Frontend benutze ich Data Grid Table und in der Tabelle brauche ich keine ID ,sondern nur Ordinalzahl.